src/api-types.ts

```diff
@@ -1040,3 +1040,28 @@ export interface LastEditedTimePropertySchema {
 export interface LastEditedByPropertySchema {
```

Comment:
This is not part of the PR, but why is this a `PropertySchema`? this shouldn't be editable right?

Reply:
Same for `  | CreatedTimePropertySchema
  | CreatedByPropertySchema
  | LastEditedTimePropertySchema
  | LastEditedByPropertySchema`

Reply:
these can be added to the database schema as properties, the values themselves are not updated

Reply:
ah i see, sg!